src/components/pages/quickstart/Prerequisites/openai.mdx

```diff
@@ -63,23 +68,9 @@ export const client = new OpenAI({
 
 </Tabs.Tab> </Tabs>
 
-<details className="rounded-lg border border-blue-100 bg-blue-50/30 p-4 my-4 dark:border-blue-900/50 dark:bg-blue-950/30 hover:bg-blue-100/50 dark:hover:bg-blue-950/40 transition-colors">
-  <summary className="flex items-center gap-2 text-sm font-medium dark:hover:bg-transparent">
-    <Info className="text-blue-500 dark:text-blue-400 w-4 h-4" />
-    <span>Keep your API key secure with environment variables</span>
-  </summary>
-
-  <div className="mt-4 text-sm text-neutral-700 dark:text-neutral-300">
-    We recommend storing your API key in a `.env` file:
-
-    1. Create a `.env` file:
-
-    ```
-    ARCADE_API_KEY=your_api_key_here
-    ```
-
-  </div>
-</details>
+<Note>
+  We recommend storing your API key in a `.env` file and using a package like `dotenv` or `python-dotenv` to load the `.env` file in your code.
+</Note>
```

Comment:
Moved this to a simpler general note, which avoids the need to have a further set of tabs inside of the expander. I think this is OK for now, unless someone finds it confusing.